gyp_xwalk

```diff
@@ -354,6 +354,18 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  parameter = '-Dverify_xwalk_apk='
```

Comment:
Nope; just like with any other variable defined with "%" at the end, a user can change the value of `verify_xwalk_apk` by passing -D to the gyp invocation (for example, `./xwalk/gyp_xwalk -Dverify_xwalk_apk=1`).